packages/taxdat/R/stan_input_helpers.R

```diff
@@ -1711,7 +1717,7 @@ impute_adm0_obs_single <- function(sf_cases_resized,
           "cases. \n")
       
     } else if (frac_coverage == 0 & nrow(ts_all_adm0) > 0) {
-      # No subnational-level data use censored national level data if available
+      # No subnational-level data use censored national level data if available 
```

Comment:
@javierps So is this an error in the comment as well? We're just imputing the maximum national censored observation regardless of whether it is censored or not

Reply:
Correct.